pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1955,17 +2126,17 @@ impl DatadirModification<'_> {
             self.put(DBDIR_KEY, Value::Image(buf.into()));
         }
         if r.is_none() {
-            // Create RelDirectory
-            // TODO: if we have fully migrated to v2, no need to create this directory
+            if v2_mode.current_status != RelSizeMigration::Legacy {
+                self.pending_directory_entries
+                    .push((DirectoryKind::RelV2, MetricsUpdate::Set(0)));
+            }
+
+            // Create RelDirectory in v1 keyspace. TODO: if we have fully migrated to v2, no need to create this directory.
```

Comment:
so this is still a TODO. When do you want to address it?

Reply:
added a code comment, currently we never set the migration state to Migrated. I'll fix it once v2 is ready and we start to deprecate v1.

(never except in a test case where the migration state had different meanings before)